src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -351,6 +352,34 @@ public void writeVariableExpr_nestedReference() {
     assertEquals(writerVisitor.write(), "x.someStringField.anotherStringField.lengthField");
   }
 
+  @Test
+  public void writeArithmeticOperationExpr_concatStringWithMethod() {
+    ValueExpr lhsExpr = ValueExpr.withValue(StringObjectValue.withValue("someWord"));
+    MethodInvocationExpr methodInvocationExpr =
+        MethodInvocationExpr.builder().setMethodName("getMethod").build();
+    MethodInvocationExpr rhsExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(methodInvocationExpr)
+            .setMethodName("getString")
+            .setReturnType(TypeNode.STRING)
+            .build();
+    ArithmeticOperationExpr arithmeticOperationExpr =
+        ArithmeticOperationExpr.concatWithExprs(lhsExpr, rhsExpr);
+    arithmeticOperationExpr.accept(writerVisitor);
+    assertThat(writerVisitor.write()).isEqualTo("\"someWord\" + getMethod().getString()");
```

Comment:
A question regarding to StringValueObject: StringValueObject is with escaper "\", but in the sample code we see "someWord + getMethod().getString()" without escaper.

Should we remove this escaper in concat case? @miraleung @xiaozhenliu-gg5

Reply:
I'm a little bit confused, what do you mean by sample code? For my understanding `String x = "someWord" + getMethod.getString();` as the test case shows is exactly we want?

Reply:
Oh, you are right. Thanks for pointing out. I am misunderstand the outside double quote.

Reply:
we are good here 👍